src/Filter/FilterComposite.php

```diff
@@ -35,6 +36,13 @@ class FilterComposite implements FilterInterface
     const CONDITION_AND = 2;
 
     /**
+     * Error message for non callable variables
+     */
+    const ERROR_MESSAGE = <<<EOT
+The value of %s should be either a callable or an instance of Zend\Stdlib\Hydrator\Filter\FilterInterface
```

Comment:
Correct.  It's also not a good idea to have duplications.  Best to be efficient.  Recommendations?